cpp/src/Ice/Makefile.mk

```diff
@@ -15,7 +15,8 @@ Ice_extra_sources       := $(filter-out src/Ice/SSL/SecureTransport%.cpp src/Ice
 endif
 
 Ice_excludes            = src/Ice/DLLMain.cpp
-Ice[shared]_excludes    = src/Ice/RegisterPluginsInit_min.cpp
+Ice[shared]_excludes    = src/Ice/AddDefaultPluginFactories_min.cpp
+Ice[static]_excludes    = src/Ice/AddDefaultPluginFactories_all.cpp
```

Comment:
Before this PR, we were including both in static builds (like the XC framework lib), and apparently picking _all.